pages/_app.tsx

```diff
@@ -30,6 +31,7 @@ type ComponentWithPageLayout = AppProps & {
 };
 
 function MyApp({ Component, pageProps }: ComponentWithPageLayout) {
+  useSession(true);
```

Comment:
This is just a question... Why is the `useSession` called here with `getSession` as true?